WooCommerce/WooCommerceTests/POS/Models/PointOfSaleAggregateModelTests.swift

```diff
@@ -262,4 +264,103 @@ struct PointOfSaleAggregateModelTests {
         // Then
         #expect(sut.itemListState == .error(expectedError))
     }
+
+    struct CartTests {
```

Comment:
Nice use of the new Swift Testing! I'll start using it more from now on.

Reply:
It's pretty nice! 

Just so you know; no need for setup/teardown, just use init/deinit, because it makes a new instance of the enclosing struct for every test that's run.